From 0a02ac779c51a2b4af3b58cb96967bf3eff80367 Mon Sep 17 00:00:00 2001
From: Wenlin Kang <wenlin.kang@windriver.com>
Date: Wed, 24 May 2017 16:45:34 +0800
Subject: [PATCH] configure: fix a cc check issue.

When has "." in cc value, the expression
$myperl -V:cc | $myperl -n -e 'print if (s/^\s*cc=.([-=\w\s\/]+).;\s*/$1/);'
can't get corretly the cc's value.

Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>

---
 configure.d/config_project_perl_python | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.d/config_project_perl_python b/configure.d/config_project_perl_python
index 475c843..22d2ad3 100644
--- a/configure.d/config_project_perl_python
+++ b/configure.d/config_project_perl_python
@@ -87,7 +87,7 @@ if test "x$install_perl" != "xno" ; then
     if test "x$enable_perl_cc_checks" != "xno" ; then
         AC_MSG_CHECKING([for Perl cc])
         changequote(, )
-        PERLCC=`$myperl -V:cc | $myperl -n -e 'print if (s/^\s*cc=.([-=\w\s\/]+).;\s*/$1/);'`
+        PERLCC=`$myperl -V:cc | $myperl -n -e 'print if (s/^\s*cc=.([-=\.\w\s\/]+).;\s*/$1/);'`
         changequote([, ])
         if test "x$PERLCC" != "x" ; then
             AC_MSG_RESULT([$PERLCC])
